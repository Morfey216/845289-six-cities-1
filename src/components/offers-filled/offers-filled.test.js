import React from 'react';
import renderer from 'react-test-renderer';

import OffersFilled from './offers-filled';

jest.mock(`../sorting/sorting`, () => `Sorting`);
jest.mock(`../offers-list/offers-list`, () => `OffersList`);
jest.mock(`../map/map`, () => `Map`);

const mockOffersData = [
  {},
  {}
];

const mockCityName = `Paris`;

it(`OffersFilled correctly renders`, () => {
  const tree = renderer
    .create(<OffersFilled
      currentCityName={mockCityName}
      currentOffersData={mockOffersData}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
