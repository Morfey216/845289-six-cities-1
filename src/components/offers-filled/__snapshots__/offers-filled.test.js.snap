// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffersFilled correctly renders 1`] = `
<div
  className="cities__places-container container"
>
  <section
    className="cities__places places"
  >
    <h2
      className="visually-hidden"
    >
      Places
    </h2>
    <b
      className="places__found"
    >
      2 places to stay in Paris
    </b>
    <Sorting
      hundlerItemClick={[Function]}
      isItemActive={false}
    />
    <OffersList
      articleClassName="cities__place-card"
      divClassName="cities__places-list places__list tabs__content"
      imageWrapperClassName="cities__image-wrapper"
      offers={
        Array [
          Object {},
          Object {},
        ]
      }
    />
  </section>
  <div
    className="cities__right-section"
  >
    <Map
      currentOffersData={
        Array [
          Object {},
          Object {},
        ]
      }
      mapClassName="cities__map"
    />
  </div>
</div>
`;
